taxcalc/policy.py

```diff
@@ -119,20 +105,55 @@ def read_json_reform(obj):
         return Parameters._read_json_revision(obj, 'policy')
 
     def implement_reform(self, reform,
```

Comment:
Not totally sure where to leave this comment, and I suspect this may be an issue in Paramtools, but when I try:

```python
salt_ref = {"ID_AllTaxes_c": {2032: [9e99, 9e99, 9e99, 9e99, 9e99]},
pol.implement_reform(salt_ref)
```

The error message is:

```
ValidationError: {
    "errors": {
        "year": "['Must be greater than {max}.']"
    }
}
```

The message should say something to the effect of "Must be <= 2030"

Reply:
Thanks @Peter-Metz  with PR https://github.com/PSLmodels/ParamTools/pull/100 this will be:

```python
ValidationError: {
    "errors": {
        "year": "['Input 2032 must be less than 2030.']"
    }
}
```